VideoWeb/VideoWeb/ClientApp/src/app/services/video-filter.service.ts

```diff
@@ -52,12 +52,27 @@ export class VideoFilterService {
         return this.filterOnSubject.asObservable();
     }
 
+    private _blurRadius = 20;
+    get blurRadius(): number {
+        return this._blurRadius;
+    }
+
+    set blurRadius(str: number) {
```

Comment:
Consider changing the name of this argument "str" might be as "string" which is a number.